Core/Utilities/File.py

```diff
@@ -4,25 +4,35 @@
    By default on Error they return None.
 """
 
-__RCSID__ = "$Id$"
-
 import os
 import hashlib as md5
 import random
 import glob
-import types
+import sys
 import re
+import errno
+
+__RCSID__ = "$Id$"
+
+def mkDir( path ):
+  try:
```

Comment:
Of course it is a problem, I totally aggree with you. But do we let the exception propagate (you know my position about it), or do we turn it into an S_ERROR ?